go.mod

```diff
@@ -437,3 +437,5 @@ require (
 	github.com/ziutek/mymysql v1.5.4 // indirect
 	golang.org/x/exp v0.0.0-20240904232852-e7e105dedf7e
 )
+
+replace go.viam.com/api => ../api
```

Comment:
To use the api defined in [this PR](https://github.com/viamrobotics/api/pull/685).